.github/workflows/main.yml

```diff
@@ -6,8 +6,23 @@ jobs:
     name: Start Job
     runs-on: ubuntu-latest
     steps:
-      - name: Start Job
-        run: echo "Start Job."
+      - name: Start Job.
```

Comment:
what's up with the various build/ci changes?

Reply:
I'm not sure what branch this was built of of originally.  I rebased it.